universal-login-react/src/ui/commons/DisplayTokens.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {IToken} from '../../core/utils/getErc721Tokens';
+import {ThemedComponent} from './ThemedComponent';
+
+interface DisplayTokenProps {
+  tokens: IToken[];
+}
+
+interface TokenProps {
+  token: IToken;
+}
+
+const Token = ({token}: TokenProps, key: string) => (
+  <ThemedComponent key={key} name="assets-item">
+    <div className="assets-tokens">
+      <table>
+        <tbody>
+          <tr className="assets-token-item-row">
+            <th className="assets-token-img-left" style={{backgroundColor: `#${token.backgroundColor}`, backgroundImage: `url(${token.image})`}}>
+            </th>
+            <th className="assets-token-item-left">
+              <div>
+                <div className="assets-one-line-paragraph assets-token-family-name">{token.tokenName}</div>
+                <p className="assets-one-line-paragraph"><a className="assets-token-name" href={token.externalink} target="_blank" rel="noopener noreferrer">{token.name}</a></p>
+              </div>
+            </th>
+            <th className="assets-token-item-right">
+              <div className="assets-token-description">{token.description}</div>
+            </th>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </ThemedComponent>
+);
+
+const DisplayTokens = ({tokens}: DisplayTokenProps) => (
+  <>
+    {tokens.length > 0 && tokens.map(token => (
```

Comment:
'map' on an empty array will return just empty array, so why we check if tokens.length > 0